ejb3/src/main/java/org/jboss/as/ejb3/subsystem/EJB3SubsystemModel.java

```diff
@@ -104,7 +104,7 @@ public interface EJB3SubsystemModel {
     String MDB_DELIVERY_GROUP_ACTIVE = "active";
 
     @Deprecated String FILE_PASSIVATION_STORE = "file-passivation-store";
-    @Deprecated String IDLE_TIMEOUT = "idle-timeout";
+    String IDLE_TIMEOUT = "idle-timeout";
```

Comment:
Revert.  The associated resource of the associated attribute is deprecated.  We can leave this alone.